### PR TITLE
Finalize 8.3 release notes

### PR DESCRIPTION
**Pull request by ljacomet**
* Contributors
* Release notes
* Release features

### CODE REVIEW

**Comment by ljacomet** [comment]
@bot-gradle test and merge

**Comment by ljacomet** [comment]
@bot-gradle test and merge

File: subprojects/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -156,21 +158,75 @@ tasks.register<Copy>("copy") {
...
+The [configuration cache](userguide/configuration_cache.html) improves build time by caching the result of the configuration phase and reusing it for subsequent builds. This feature can significantly improve build performance.
+
+#### Temporarily opting out of stricter configuration inputs detection
+
+In order to ensure correctness, configuration cache detects undeclared inputs when plugins or build logic perform file system checks such as `File.exists()` or `File.isFile()`.
+However, not all plugins might be updated yet to properly declare all the inputs, leading to cache misses and suboptimal performance.
+
+Configuration cache now has an opt-out option for temporarily ignoring such undeclared inputs.
+
+This option mitigates undesired configuration cache misses, until the plugins are properly updated.
```
Comment (lkasso) [inline subprojects/docs/src/docs/release/notes.md:180]:
```suggestion
This option mitigates undesired configuration cache misses until the plugins are properly updated.
```

File: subprojects/docs/src/docs/release/notes.md [hunk 2]
```diff
@@ -1,6 +1,16 @@
 The Gradle team is excited to announce Gradle @version@.
 
-This release features [1](), [2](), ... [n](), and more.
+This release features the support for [persistent Java compiler daemons](#faster-java-compilation) to speed up Java compilation.
+Gradle will also use [less memory for dependency resolution](#reduced-memory-consumption).
+The effect is significant for large builds, in particular, large Android builds including, but not limited to, IDE sync.
```
Comment (lkasso) [inline subprojects/docs/src/docs/release/notes.md:5]:
```suggestion
The effect is significant, particularly for large builds such as Android builds.```

Comment (lkasso) [inline subprojects/docs/src/docs/release/notes.md:5]:
```suggestion
The effect is significant, particularly for large builds such as Android builds.
```

Comment (lkasso) [reply subprojects/docs/src/docs/release/notes.md:5]: I don't see a way to add "IDE sync". It's just a random add at this point.

Comment (ljacomet) [reply subprojects/docs/src/docs/release/notes.md:5]: Thanks, we have the specifics in the section. Making it shorter for the preamble makes sense IMO

**Comment by ljacomet** [comment]
@bot-gradle test and merge

**Pull request merged**